src/main/java/io/strimzi/kafka/metrics/KafkaPrometheusMetricsReporter.java

```diff
@@ -57,7 +56,7 @@ public void configure(Map<String, ?> map) {
         config = new PrometheusMetricsReporterConfig(map, registry);
         collector = new KafkaMetricsCollector();
         // Add JVM metrics
-        JvmMetrics.builder().register(registry);
+        JvmMetrics.builder().register();
```

Comment:
The method without an argument is idempotent and registers the JVM metrics onto the default metrics registry, which is what we use at runtime. This allow multiple components to run this code without causing exceptions as the metrics are only added once.

It would be nice to pass a registry and still have the idempotent behavior if it's the default registry. I may open a PR on https://github.com/prometheus/client_java to do that. In the meantime, we have to use to method without arguments.